components/tools/OmeroPy/test/unit/clitest/test_db.py

```diff
@@ -136,14 +136,19 @@ def testScript(self, pos_input, no_salt, file_arg):
             output = self.file
         else:
             output = self.script_file
-        if "version" not in pos_input or "patch" not in pos_input:
-            self.expectVersion(self.data["version"])
-            self.expectPatch(self.data["patch"])
```

Comment:
I suppose the two methods above are now obsolete and can be removed from the test class.